test/cufinufft2d1_test.cu

```diff
@@ -141,7 +151,7 @@ int main(int argc, char* argv[])
 		cudaMemcpyDeviceToHost));
 
 	printf("[Method %d] %ld NU pts to #%d U pts in %.3g s (\t%.3g NU pts/s)\n",
-			dplan.opts.gpu_method,M,N1*N2,totaltime/1000,M/totaltime*1000);
+			dplan->opts.gpu_method,M,N1*N2,totaltime/1000,M/totaltime*1000);
```

Comment:
`opts.gpu_method` should suffice here, right? Don't want to be dereferencing `dplan` unless we absolutely have to.

Reply:
Correct, good point.

Reply:
There are a few more occurrences of this, otherwise we're good here.

Reply:
I fixed the two remaining in tests.  There are many refs internally in `src/` but the bunch I looked at appear to be cases where the function is only passed a `CUFINUFFT_PLAN` (no other opts struct).

Reply:
That's totally fine. Just the public API stuff that we should be careful with here.